codex-rs/tui/src/history_cell.rs

```diff
@@ -466,24 +467,34 @@ impl ExecCell {
                 body_lines.extend(wrapped_borrowed.iter().map(|l| line_to_static(l)));
             }
         }
-        if let Some(output) = call.output.as_ref()
-            && output.exit_code != 0
-        {
-            let out = output_lines(
-                Some(output),
-                OutputLinesParams {
-                    only_err: false,
-                    include_angle_pipe: false,
-                    include_prefix: false,
-                },
-            )
-            .into_iter()
-            .join("\n");
-            if !out.trim().is_empty() {
-                // Wrap the output.
-                for line in out.lines() {
-                    let wrapped = textwrap::wrap(line, TwOptions::new(width as usize - 4));
-                    body_lines.extend(wrapped.into_iter().map(|l| Line::from(l.to_string().dim())));
+
+        if let Some(output) = call.output.as_ref() {
+            let is_shell_exec_cell = self.calls.iter().any(|c| c.user_initiated_shell_command);
```

Comment:
Do we believe we could have a mix of exec calls in flight, only some of which are user initiated shell commands?